evap/static/scss/components/_confirmation-modal.scss

```diff
@@ -0,0 +1,82 @@
+confirmation-modal:not(:defined) > :not([slot="submit-group"]) {
+  // Without this, the elements that are slotted into the dialog element show up as if they are just normal child
+  // elements and disappear once the custom element registration is done. To avoid a short flicker of these elements,
+  // we hide them until the constructor of the custom element has run.
+  display: none;
+}
+
+dialog.evap-modal-dialog {
+  z-index: $zindex-modal;
+  inset-block-start: -70vh; // moves the modal to the top of the page, by default, it is centered
```

Comment:
The delegation modal looks broken, if zoom >= 170% for me. Other modals also move up, when zoom increases.
Maybe something with `top` is better (or needed in addition)?